tools/releases/dockerfiles/Dockerfile.kuma-injector

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.10
```

Comment:
Docker image should run under a non-`root` account, e.g. `nobody` (`65534:65534`) on `alpine`